src/Test/TestCases.Workflows/DataContractSerializerTests.cs

```diff
@@ -1,32 +1,52 @@
-﻿using Shouldly;
+﻿using JsonFileInstanceStore;
+using Newtonsoft.Json;
+using Shouldly;
 using System.Activities;
 using System.Activities.Statements;
 using System.Activities.XamlIntegration;
 using System.Collections.Generic;
 using System.IO;
 using System.Runtime.Serialization;
 using Xunit;
+
 namespace TestCases.Workflows
 {
     public class DataContractSerializerTests

```

Comment:
rename to not refer to DataContract in the class name

Reply:
Renamed class to `CompiledLocationSerializationTests`. Fixed.